parlai/tasks/cornell_movie/agents.py

```diff
@@ -63,20 +58,25 @@ def rebuild(entries):
                         for i in range(len(entries) - 1)
                     ]
                 )
-                if len(entries[-1]) > 1 and entries[-1][1]:
-                    # add y_n => '', if last y avail
-                    new_list.append((entries[-1][1][0], None))
             return new_list
 
+        def is_valid(entry):
```

Comment:
okay, since you're over here anyway, can you move rebuild and is_valid to be part of the class instead of closures?, i.e. `def _rebuild(self, entries)` and `def _is_valid(self, entry)`?

Reply:
Done